xbmc/cores/VideoPlayer/VideoReferenceClock.cpp

```diff
@@ -206,7 +202,7 @@ int64_t CVideoReferenceClock::GetTime(bool interpolated /* = true*/)
   }
   else
   {
-    return CurrentHostCounter() + m_ClockOffset;
```

Comment:
Heh ok... just removed it and everything worked fine, so I assumed it's some leftover code. Also it didn't really make any sense.